guides/micronaut-jms-aws-sqs/groovy/src/main/groovy/example/micronaut/Application.groovy

```diff
@@ -0,0 +1,11 @@
+package example.micronaut
```

Comment:
delete all 3 `Application` classes - these are generated with the rest of the initial app. You only need to add new or modified source files

Reply:
Thank you. That's good to know.